docs/polaris/workflows/parsl.md

```diff
@@ -108,9 +108,8 @@ config = Config(
 
 ## Special notes for Polaris
 
-On Polaris, we are currently investigating a hang that occurs in some parallel applications.  The hang occurs under these circumstances, as far as we can tell at this time:
-- The application is launched on multiple nodes using `mpi`.
-- The application uses ``fork`` on each node to spawn processes
-- (Likely, but unconfirmed) the application has some aspect of locking involved, perhaps not visible to the user.
+On Polaris, there is a known bug where python applications launched with `mpi` and that use ``fork`` to spawn processes can sometimes have unexplaned hangs.  For this reason, it is recommended to use ``start_method="spawn"`` on Polaris when using the ``MpiExecLauncher`` as is shown in the example config above.  Alternatively, another solution is to use the ``GNUParallelLauncher`` which uses ``GNU Parallel`` to spawn processes.  ``GNU Parallel`` can be loaded in your environment with the command ``module load gnu-parallel``.  Both of these approaches will circumvent the hang issue from using ``fork``.
+
+## Updates
 
-Under these circumstances, we have observed that rank 0 will proceed as far as possible, but ranks 1 to N will hang.  For ``parsl``, which uses ``python.multiprocessing`` (which calls ``fork``), hangs can occur for some workloads on remote ranks.  In this case, you can install ``GNU Parallel`` instead and use that as a launcher in ``parsl`` - you can install it into the virtual env as parsl is involved above.  This will circumvent the hang issue if your ``parsl`` workloads require ``fork`` calls.
+For ``parsl`` versions after July 2023, the ``address`` passed in the ``HighThroughputExecutor`` needs to be set to ``address = address_by_interface("bond0")``.  With ``parsl`` versions prior to July 2023, it was recommended to use ``address = address_by_hostname()`` on Polaris, but with later versions this will not work on Polaris (or any other machine).
```

Comment:
minor quibble, but why are you using double backticks for inline code blocks? I would recommend using single backticks, unless you need a backtick in the code